cmdutil/doc.go

```diff
@@ -0,0 +1,94 @@
+// Package cmdutil contains helper utilities for setting up a CLI with Go,
+// providing basic application behavior and for reducing boilerplate code.
+//
+// An example application can be found at
+// https://github.com/rebuy-de/golang-template.
+//
+// Graceful Application Exits
+//
+// In many command line applications it is desired to exit the process
+// immediately, if it is clear that the application cannot recover. Important
+// note: This is designed for actual applications (ie not libraries), because
+// only the application itself should decide when to exit. Libraries should
+// alway return errors.
+//
+// There are three ways to handle fatal errors in Go. With os.Exit() the
+// process will terminate immediately, but it will not call any deferrers which
+// means that possible cleanup task do not get called. The next way is to call
+// panic, which respects the defer statements, but unfortunately it is not
+// possible to define an exit code and the user gets confused with a stack
+// trace. Finally, the function could just return an error indicating that
+// things failed, but this intriduces a lot of code and conditionals and
```

Comment:
*introduces

Reply:
[...] lot of code, conditionals and [...]

Reply:
fixed